examples/feedback.rs

```diff
@@ -81,6 +87,7 @@ fn main() -> Result<(), coreaudio::Error> {
             mut data,
             ..
         } = args;
+        println!("input cb {} frames", num_frames);
```

Comment:
Maybe we should avoid `println`s in the audio callbacks? Same goes for some of the other examples. I think it's generally good practise to avoid any sort of I/O on real-time threads, but I do see how it can be useful and harmless in examples.

Alternatively, maybe even just a comment saying that they're to help gain an intuition for how streams work and that you should avoid this in practise.

Reply:
Having these prints here were quite helpful for my own understanding. I added a comment to not do this in a real application.